clients/drcachesim/tools/func_view.h

```diff
@@ -93,6 +93,7 @@ class func_view_t : public analysis_tool_t {
     };
     struct shard_data_t {
         memref_tid_t tid = 0; // We only support SHARD_BY_THREAD.
+        uint64_t filetype;
```

Comment:
Add a default safety value so this won't be uninitialized.

Reply:
Done.